src/commands/list.js

```diff
@@ -31,21 +31,23 @@ export const actionCreator = () => async ({
 	inputs,
 	options: { testnet },
 }) => {
-	const singularType = Object.keys(SINGULARS).includes(type)
-		? SINGULARS[type]
-		: type;
+	const pluralType = Object.keys(PLURALS).includes(type) ? PLURALS[type] : type;
 
-	if (!COMMAND_TYPES.includes(singularType)) {
+	if (!COMMAND_TYPES.includes(pluralType)) {
 		throw new ValidationError('Unsupported type.');
 	}
 
-	const queries = inputs.map(input =>
-		query.handlers[deAlias(singularType)](input, { testnet }),
-	);
+	const endpoint = deAlias(pluralType);
 
-	return Promise.all(queries).then(results =>
-		results.map(processQueryResult(singularType)),
-	);
+	const queries = inputs.map(input => {
+		const req = {
+			limit: 1,
```

Comment:
Why do we need this for `list`?

Reply:
it's because there is no way to specify array or multiple ids on the Core API.
Therefore we need multiple requests